packages/pages/src/util/resolveApiBase.ts

```diff
@@ -11,19 +11,23 @@ const API_BASE_EU = "https://api.eu.yext.com/v2/accounts/me/";
  * @returns The API base URL string
  * @throws Error if EU partition is used with a non-production environment
  */
-export function resolveApiBase(partition: "US" | "EU", env: string): string {
+export function resolveApiBase(
+  partition: "US" | "EU",
+  env: "prod" | "production" | "sbx" | "sandbox"
+): string {
   // Normalize environment values
   const isProduction = env === "prod" || env === "production";
   const isSandbox = env === "sbx" || env === "sandbox";
 
-  switch (partition) {
-    case "EU":
-      if (!isProduction) {
-        throw new Error("EU partition only supports production environment");
-      }
-      return API_BASE_EU;
-    case "US":
-    default:
-      return isSandbox ? API_BASE_SBX : API_BASE_PROD;
+  if (isProduction) {
```

Comment:
I still think it's better to use switch cases so it's easy to add new ones in the future